package.json

```diff
@@ -1,63 +1,64 @@
 {
-  "name": "@story-protocol/protocol-core",
-  "version": "v1.0.0-beta-rc6",
-  "description": "Story Protocol core smart contracts",
-  "main": "",
-  "directories": {
-    "lib": "lib",
-    "test": "test"
-  },
-  "author": "StoryProtocol",
-  "license": "UNLICENSED",
-  "scripts": {
-    "lint": "npm run lint:js && npm run lint:sol",
-    "lint:fix": "npm run lint:js:fix && npm run lint:sol:fix",
-    "lint:js": "prettier --log-level warn --ignore-path .gitignore '**/*.{js,ts}' --check && eslint --ignore-path .gitignore .",
-    "lint:js:fix": "prettier --log-level warn --ignore-path .gitignore '**/*.{js,ts}' --write && eslint --ignore-path .gitignore . --fix",
-    "lint:sol": "prettier --log-level warn --ignore-path .gitignore '{contracts,test}/**/*.sol' --check && solhint '{contracts,test}/**/*.sol'",
-    "lint:sol:fix": "prettier --log-level warn --ignore-path .gitignore '{contracts,test}/**/*.sol' --write",
-    "solhint": "solhint '{contracts,test}/**/*.sol'",
-    "test": "npx hardhat test",
-    "prepare": "husky install",
-    "docgen": "hardhat docgen"
-  },
-  "devDependencies": {
-    "@nomicfoundation/hardhat-ethers": "^3.0.5",
-    "@nomicfoundation/hardhat-foundry": "^1.1.1",
-    "@nomicfoundation/hardhat-verify": "^2.0.3",
-    "@openzeppelin/hardhat-upgrades": "^3.0.2",
-    "@tenderly/hardhat-tenderly": "^2.2.1",
-    "@typechain/ethers-v6": "^0.5.1",
-    "@typechain/hardhat": "^9.1.0",
-    "base64-sol": "^1.1.0",
-    "chai": "^5.0.3",
-    "dotenv": "^16.4.1",
-    "ds-test": "https://github.com/dapphub/ds-test",
-    "eslint": "^8.56.0",
-    "eslint-plugin-prettier": "^5.1.3",
-    "ethers": "^6",
-    "forge-std": "github:foundry-rs/forge-std#v1.7.6",
-    "hardhat": "^2.19.4",
-    "hardhat-contract-sizer": "^2.10.0",
-    "hardhat-deploy": "^0.11.45",
-    "hardhat-deploy-ethers": "^0.4.1",
-    "hardhat-gas-reporter": "^1.0.10",
-    "husky": "^8.0.0",
-    "minimatch": "^9.0.3",
-    "mocha": "^10.2.0",
-    "prettier": "^3.0.0",
-    "prettier-plugin-solidity": "^1.1.0",
-    "solhint": "^4.1.1",
-    "solhint-community": "^3.7.0",
-    "solhint-plugin-prettier": "^0.1.0",
-    "solidity-coverage": "^0.8.6",
-    "solidity-docgen": "^0.6.0-beta.36",
-    "ts-node": "^10.9.2",
-    "typechain": "^8.3.2"
-  },
-  "dependencies": {
-    "@openzeppelin/contracts": "5.0.2",
-    "@openzeppelin/contracts-upgradeable": "5.0.2",
-    "erc6551": "^0.3.1"
-  }
+    "name": "@story-protocol/protocol-core",
```

Comment:
why do you need to update this file? I didn't see much changes here.

Reply:
We added "@openzeppelin/contracts-upgradeable-v4" - that was the change in that file. The reason the whole document is marked as changing is because of spacing (format) changes

Reply:
can we have a uniformed space setting in our linter?  I don't want to review the whitespace changes. Not an urgent, but it is annoying. @jdubpark